test/functional/tests/fault_injection/test_fault_injection_core_in_raid.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import re
+import pytest
+from api.cas import casadm, cli_messages
+from api.cas.cache_config import CacheMode
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from storage_devices.raid import Raid, RaidConfiguration, MetadataVariant, Level
+from test_utils.size import Size, Unit
+from api.cas.cli_messages import (
+    partition_not_suitable_for_array,
+    device_or_resource_busy
+)
+
+expected_msg_1 = partition_not_suitable_for_array[0]
+expected_msg_2 = device_or_resource_busy[0]
+
+
+@pytest.mark.parametrizex("cache_mode", CacheMode)
```

Comment:
Different cache modes don't affect this scenario so I suggest getting rid of iterating over them. Cache may run in WT or WB